Obsidian/Services/IUserCache.cs

```diff
@@ -30,7 +30,17 @@ public sealed class UserCache(HttpClient httpClient, ILogger<UserCache> logger)
         if (cachedUser != null && !cachedUser.Expired)
             return cachedUser;
 
-        var user = await httpClient.GetFromJsonAsync<MojangProfile>($"{userWithNameEndpoint}{escapedUsername}", Globals.JsonOptions);
+        MojangProfile? user;
+
+        try
+        {
+            user = await httpClient.GetFromJsonAsync<MojangProfile>($"{userWithNameEndpoint}{escapedUsername}", Globals.JsonOptions);
+        }
+        catch
+        {
+            user = null;
+            logger.LogWarning("Could not get user profile for {Username}", username);
+        }
 
         if (user is null)
             return null;
```

Comment:
Return the response in the try-catch instead of throwing it on a variable outside and just return null outside

Reply:
Good now?